kubectl-plugin/pkg/util/generation/generation.go

```diff
@@ -16,45 +22,72 @@ import (
 	rayv1ac "github.com/ray-project/kuberay/ray-operator/pkg/client/applyconfiguration/ray/v1"
 )
 
-type RayClusterSpecObject struct {
-	HeadRayStartParams     map[string]string
-	WorkerRayStartParams   map[string]string
-	HeadNodeSelectors      map[string]string
-	WorkerNodeSelectors    map[string]string
-	RayVersion             string
-	Image                  string
-	HeadCPU                string
-	HeadGPU                string
-	HeadMemory             string
-	HeadEphemeralStorage   string
-	WorkerCPU              string
-	WorkerGPU              string
-	WorkerMemory           string
-	WorkerEphemeralStorage string
-	WorkerReplicas         int32
-}
-
-type RayClusterYamlObject struct {
-	ClusterName string
-	Namespace   string
-	Labels      map[string]string
-	Annotations map[string]string
-	RayClusterSpecObject
+const (
+	volumeName = "cluster-storage"
```

Comment:
I hardcoded the name using the same one in [Distributed checkpointing with KubeRay and GCSFuse](https://docs.ray.io/en/latest/cluster/kubernetes/examples/distributed-checkpointing-with-gcsfuse.html). Lmk if this should be exposed to user and configurable.